assets/js/modules/analytics/components/dashboard/DashboardUniqueVisitorsWidget.stories.js

```diff
@@ -0,0 +1,215 @@
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( options ), { options } );
+		dispatch( STORE_NAME ).startResolution( 'getReport', [ options ] );
+		dispatch( STORE_NAME ).startResolution( 'getReport', [ optionsCompare ] );
+	},
+};
+
+export const DataUnavailable = WidgetAreaTemplate.bind();
+DataUnavailable.storyName = 'Data Unavailable';
+DataUnavailable.args = {
+	setupRegistry: ( { dispatch } ) => {
+		dispatch( STORE_NAME ).receiveGetReport( [], { options } );
+	},
+};
+
+export const Error = WidgetAreaTemplate.bind();
+Error.storyName = 'Error';
+Error.args = {
+	setupRegistry: ( { dispatch } ) => {
+		const error = {
+			code: 'missing_required_param',
+			message: 'Request parameter is empty: metrics.',
+			data: {},
+		};
+		dispatch( STORE_NAME ).receiveError( error, 'getReport', [ options ] );
+		dispatch( STORE_NAME ).finishResolution( 'getReport', [ options ] );
+	},
+};
+
+export const LoadedEntityURL = WidgetAreaTemplate.bind();
+LoadedEntityURL.storyName = 'Loaded with entity URL set';
+LoadedEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsCompareEntityURL ), { options: optionsCompareEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsEntityURL ), { options: optionsEntityURL } );
+	},
+};
+
+export const LoadingEntityURL = WidgetAreaTemplate.bind();
+LoadingEntityURL.storyName = 'Loading with entity URL set';
+LoadingEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsCompareEntityURL ), { options: optionsCompareEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsEntityURL ), { options: optionsEntityURL } );
+		dispatch( STORE_NAME ).startResolution( 'getReport', [ optionsEntityURL ] );
+		dispatch( STORE_NAME ).startResolution( 'getReport', [ optionsCompareEntityURL ] );
+	},
+};
+
+export const DataUnavailableEntityURL = WidgetAreaTemplate.bind();
+DataUnavailableEntityURL.storyName = 'Data Unavailable with entity URL set';
+DataUnavailableEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( [], { options: optionsEntityURL } );
+	},
+};
+
+export const ErrorEntityURL = WidgetAreaTemplate.bind();
+ErrorEntityURL.storyName = 'Error with entity URL set';
+ErrorEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+		const error = {
+			code: 'missing_required_param',
+			message: 'Request parameter is empty: metrics.',
+			data: {},
+		};
+
+		provideSiteInfo( registry, { currentEntityURL } );
```

Comment:
This line looks like it belongs in the global decorator, unless there's something about it which is specific to the error case here.

Reply:
The error message/layout will be the same, but it's just making sure it displays correctly when an entity URL is set.